### ISSUE TITLE
Image captions

### ISSUE THREAD

**Issue by benseverett** [arctheowl/snezhi-portfolio#9]
Can the some of the images that pop out of the sliders have captions?

**Comment by arctheowl** [comment]
Yes, it can either be a title on the pop out, so at the top, or a caption at the bottom over the image
